prisma/migrations/20240426110344_add_new_status/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterEnum
+ALTER TYPE "MembershipStatus" ADD VALUE 'WAITING';
```

Comment:
Why don't we use pending ?

Reply:
We have to think about it... I am not sure also it is the right name, but it is something like, when you create a membership it isn't paid yet. So, I think a different status just to differentiate from who's paid and its waiting for approval and other users that they didn't have finished their payment

Reply:
I'm afraid of having orphans member in these scenario. It could be helpful to have a column per member to identify the payment status as it could be unrelated from the member status as it is a consequence (especially in a renewal case).

To me the waiting status is an over head

Reply:
We have `lastPaymentAt` which ensures the subscription has been successfully paid... I don't know what can be better when we operate with those users.